tensorflow/lite/kernels/batch_matmul_test.cc

```diff
@@ -553,6 +558,34 @@ TEST_P(QuantizedBatchMatMulOpTest, SimpleTestQuantizedInt8) {
   EXPECT_THAT(m.GetOutput<int8_t>(), ElementsAre(22, 22, 22, 56, 56, 56));
 }
 
+TEST_P(QuantizedBatchMatMulOpTest, SimpleTestQuantizedInt16) {
+  const float inputs_scale = 2.0 * 10 / std::numeric_limits<uint16_t>::max();
+  const float output_scale = 1.0;
+
+  QuantizedBatchMatMulOpModel m(
+      /*units=*/3, /*batches*/ 2,
+      /*lhs=*/
+      {TensorType_INT16, {2, 10}, 0, 0, inputs_scale, 0},
```

Comment:
I currently set the input scale and zero-point manually as it seems that the `QuantizationParams` method in `test_utils.h` called by `AddTensor` returns a zero-point of -1 when I pass a symmetric range but the int16x8 BATCH_MATMUL kernel is restricted to a zero-point of 0. 

For example calling `QuantizationParams<int16_t>(-10.0, 10.0)` returns a -1 zero-point, is it an expected behaviour?